lib/nfsbase.pm

```diff
@@ -0,0 +1,146 @@
+package nfsbase;
+use mmapi;
+use testapi;
+use strict;
+use warnings;
+use utils;
+use mm_network;
+use version_utils;
+
+
+our @ISA    = qw(Exporter);
+our @EXPORT = qw(server_configure_network try_nfsv2 prepare_exports yast_handle_firewall add_rw_share mount_export client_common_tests);
+
+sub server_configure_network {
+    my ($self) = @_;
+
+    # Configure static IP for client/server test
+    configure_default_gateway;
+    configure_static_ip('10.0.2.101/24');
+    configure_static_dns(get_host_resolv_conf());
+
+    if (is_sle('15+')) {
+        record_soft_failure 'boo#1130093 No firewalld service for nfs-kernel-server';
+        systemctl 'stop ' . $self->firewall;
+        systemctl 'disable ' . $self->firewall;
+    }
+}
+
+sub try_nfsv2 {
+    # Try that NFSv2 is disabled by default
+    systemctl 'restart nfsserver';
+    assert_script_run "cat /proc/fs/nfsd/versions | grep '\\-2'";
```

Comment:
not that you version is wrong but just showing you other options , you may leave your version 
```suggestion
    validate_script_output( "cat /proc/fs/nfsd/versions",  sub { /-2/ } );
```

Reply:
I know this one too and the code is simpler but in my opinion this is harder to read from the actual test run. I don't know then.

Reply:
yeah sure , as I said your version will work to . so no problem to keep it